{
	"name": "CVE-2015-0219",
	"threat_severity": "Moderate",
	"public_date": "2015-01-13T00:00:00Z",
	"bugzilla": {
		"description": "Django: WSGI header spoofing via underscore/dash conflation",
		"id": "1179672",
		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1179672"
	},
	"cvss": {
		"cvss_base_score": "5.8",
		"cvss_scoring_vector": "AV:N/AC:M/Au:N/C:P/I:P/A:N",
		"status": "draft"
	},
	"cwe": "(CWE-20|CWE-290)",
	"details": [
		"Django before 1.4.18, 1.6.x before 1.6.10, and 1.7.x before 1.7.3 allows remote attackers to spoof WSGI headers by using an _ (underscore) character instead of a - (dash) character in an HTTP header, as demonstrated by an X-Auth_User header."
	],
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2015-0219\nhttps://nvd.nist.gov/vuln/detail/CVE-2015-0219\nhttps://www.djangoproject.com/weblog/2015/jan/13/security/"
	],
	"acknowledgement": "Red Hat would like to thank upstream Django project for reporting this issue.",
	"package_state": [
		{
			"cpe": "cpe:/a:redhat:openstack:5::el6",
			"fix_state": "Not affected",
			"package_name": "python-django",
			"product_name": "Red Hat Enterprise Linux OpenStack Platform 5 (Icehouse)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:6",
			"fix_state": "Not affected",
			"package_name": "python-django",
			"product_name": "Red Hat Enterprise Linux OpenStack Platform 6 (Juno)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:4",
			"fix_state": "Not affected",
			"package_name": "Django14",
			"product_name": "Red Hat OpenStack Platform 4"
		},
		{
			"cpe": "cpe:/a:rhel_sam:1",
			"fix_state": "Not affected",
			"package_name": "Django",
			"product_name": "Red Hat Subscription Asset Manager"
		}
	],
	"upstream_fix": "Django 1.4.18, Django 1.6.10, Django 1.7.3",
	"csaw": false
}
